.github/workflows/ui.yaml

```diff
@@ -108,20 +108,20 @@ jobs:
       - name: Scan image with Trivy
         uses: aquasecurity/trivy-action@master
         with:
-          image-ref: '${{ env.REGISTRY }}:${{ env.BRANCH }}'
-          format: 'sarif'
-          output: 'trivy-results.sarif'
+          image-ref: "${{ env.REGISTRY }}:${{ env.BRANCH }}"
+          format: "sarif"
+          output: "trivy-results.sarif"
           ignore-unfixed: true
-          severity: 'CRITICAL,HIGH'
-          exit-code: '1'
+          severity: "CRITICAL,HIGH"
+          exit-code: "1"
         env:
           TRIVY_USERNAME: ${{ github.actor }}
           TRIVY_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
       - name: Upload Trivy scan results to GitHub Security tab
         if: always()
         uses: github/codeql-action/upload-sarif@v2
         with:
-          sarif_file: 'trivy-results.sarif'
+          sarif_file: "trivy-results.sarif"
   deploy:
     if: ${{ github.actor != 'dependabot[bot]' }} # Don't deploy Dependabot changes
     runs-on: ubuntu-latest
```

Comment:
Something to watch will be if Prettier starts trying to update the GitHub Action Literal here. I discovered double quotes aren't valid in literals in https://github.com/NOAA-GSL/unified-graphics/commit/3334819249c331a42b76c9dfac41cd078625a8ad. Thus far Prettier's done a good job of detecting them.

Reply:
I reverted all of the double quotes to single quotes, though it doesn’t look to me like anything is running.

Reply:
Hmmm... I'm not seeing anything running either.

Oh. Gosh. I forgot that stacked PRs don't target `main` so aren't run in CI. 🤦‍♂️ My apologies. It may make sense to try updating the CI files to remove the `branches` filter in the `on.pull_request` part of the API, UI, and data CI files so we get CI for stacked PRs:

```diff
on:
  ...
  pull_requests:
    ...
-    branches: [main]
```

Otherwise, CI won't run either way so we could go ahead and merge this and deal with that later.

Reply:
Ah, oops. I thought we’d made that change separately. 😳

Reply:
Me too. I wonder if I did the change on MATS and didn't do it here as well.